src/services/activityReports.js

```diff
@@ -3,13 +3,34 @@ import { Op } from 'sequelize';
 
 import {
   ActivityReport,
+  ActivityReportCollaborator,
   sequelize,
   ActivityRecipient,
   Grant,
   Grantee,
   NonGrantee,
+  User,
 } from '../models';
 
+async function saveReportCollaborators(activityReportId, collaborators, transaction) {
+  await ActivityReportCollaborator.destroy({
```

Comment:
Would doing this lose information about when a collaborator was initially added (if they were there before, were removed, than re-added below)? Would an UPSERT followed by a delete of removed collaborators might be slightly better, or not work for reasons that aren't apparent to me?

Reply:
No you are right upsert is more appropriate here. I'll update (and probably update another function that does something similar)